fastapi_sqla/utils.py

```diff
@@ -0,0 +1,21 @@
+import os
+
+import boto3
+
+ENDPOINT = os.getenv("RDS_ENDPOINT", "")
+REGION = os.getenv("AWS_REGION", "")
```

Comment:
```suggestion
REGION = os.getenv("AWS_DEFAULT_REGION", "")
```
Default region is the one injected by the cluster already and normally picked up by BOTO automatically